dealroom_firestore_connector/__init__.py

```diff
@@ -375,7 +375,7 @@ def set_history_doc_refs(
 
     if history_refs == ERROR:
         return ERROR
-    if "dealroom_id" in history_refs:
+    if "dealroom_id" in history_refs and len(history_refs["dealroom_id"]) > 0:
         count_history_refs = len(history_refs["dealroom_id"])
         key_found = "dealroom_id"
     elif "dealroom_id_old" in history_refs:
```

Comment:
Wouldn't it enter here and retrieve an error if we are matching by dealroom_id_old?

Reply:
No, the result from history_refs when matching by a number is something like:
```python
{
    "dealroom_id": [],
    "dealroom_id_old": [kuhbcaksdjhasukh], <-- doc ref
} 
```

Reply:
Updated in 6237af2